instrumentation/dubbo/dubbo-apache-2.7/library/src/main/java/io/opentelemetry/instrumentation/dubbo/apache/v2_7/common/DubboHelper.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.dubbo.apache.v2_7.common;
+
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.StatusCode;
+import io.opentelemetry.semconv.trace.attributes.SemanticAttributes;
+import org.apache.dubbo.rpc.Result;
+
+public final class DubboHelper {
+
+  private DubboHelper() {}
+
+  public static void prepareSpan(Span span, String methodName, String interfaceName) {
```

Comment:
Order it `interfaceName`, then `methodName` which is more common for class+method